addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -3,17 +3,15 @@ import { usePos } from "@point_of_sale/app/store/pos_hook";
 import { Component, onMounted, onWillUnmount, useState } from "@odoo/owl";
 import { Dialog } from "@web/core/dialog/dialog";
 import { useService } from "@web/core/utils/hooks";
-import {
-    getTaxesAfterFiscalPosition,
-    getTaxesValues,
-} from "@point_of_sale/app/models/utils/tax_utils";
 
 export class ScaleScreen extends Component {
```

Comment:
could you make this not depend on pos? This way you could reuse it directly in the customer display. It would cut on duplication.

It would have to be a simple component with no logic inside. You would compute everything you need outside ( in PosStore for ex ) and just pass props to it



Also we should rename it to something like `ScaleDialog`, as it is no longer a screen